web/renderer/components/DeleteModal/index.tsx

```diff
@@ -15,10 +15,11 @@ type Props<TData, TVariables> = {
   assetId?: string;
   btnText?: string;
   mutationProps: MutationProps<TData, TVariables>;
-  callback?: (d: TData) => void;
+  callback?: (d: TData) => Error | void;
```

Comment:
This should probably be `Error | undefined`

Another way to do this would be to pass an optional `setErr` to `callback` instead of returning an error here